Src/Support/GoogleApis.Core/Apis/Http/ConfigurableMessageHandler.cs

```diff
@@ -173,6 +174,19 @@ public void RemoveExecuteInterceptor(IHttpExecuteInterceptor interceptor)
 
         #endregion
 
+        private int _loggingRequestId = 0;
+
+        private ILogger _instanceLogger = Logger;
+
+        /// <summary>
+        /// For testing only

```

Comment:
I generally follow this: https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md
Which specifies leading _ for instance fields.
This repo certainly doesn't generally follow this style, but personally I prefer to follow it for new code.